client/components/mma/cancel/cancellationSaves/digipack/ThankYouOffer.tsx

```diff
@@ -0,0 +1,180 @@
+import {
+	Button,
+	buttonThemeReaderRevenueBrand,
+	Stack,
+	SvgTickRound,
+} from '@guardian/source-react-components';
+import { useContext } from 'react';
+import { Navigate, useLocation, useNavigate } from 'react-router';
+import {
+	buttonCentredCss,
+	buttonContainerCss,
+} from '@/client/styles/ButtonStyles';
+import { dateString } from '../../../../../../shared/dates';
+import { benefitsCss } from '../../../shared/benefits/BenefitsStyles';
+import { Heading } from '../../../shared/Heading';
+import type {
+	CancellationContextInterface,
+	CancellationRouterState,
+} from '../../CancellationContainer';
+import { CancellationContext } from '../../CancellationContainer';
+
+const DiscountOffer = ({
+	handleDiscountOfferClick,
+}: {
+	handleDiscountOfferClick: () => void;
+}) => (
+	<Stack
+		space={4}
+		css={css`
+			background-color: #f3f7fe;
+			border-radius: 4px;
+			padding: ${space[4]}px;
+		`}
+	>
+		<div>
+			<div
+				css={css`
+					${textSans.large({ fontWeight: 'bold' })}
+					margin-bottom: ${space[2]}px;
+				`}
+			>
+				A subscription offer just for you
+			</div>
+			<ul css={benefitsCss}>
+				<li>
+					<SvgTickRound isAnnouncedByScreenReader size="medium" />
+					<span
+						css={css`
+							padding-top: ${space[1]}px;
+						`}
+					>
+						Get a 25% discount for 3 months (x, then y)
+					</span>
+				</li>
+				<li>
+					<SvgTickRound isAnnouncedByScreenReader size="medium" />{' '}
+					<span
+						css={css`
+							padding-top: ${space[1]}px;
+						`}
+					>
+						Keep all your supporter extras, including unlimited,
+						ad-free reading
+					</span>
+				</li>
+				<li>
+					<SvgTickRound isAnnouncedByScreenReader size="medium" />{' '}
+					<span
+						css={css`
+							padding-top: ${space[1]}px;
+						`}
+					>
+						Retain exclusive access to our daily Editions app
+					</span>
+				</li>
+			</ul>
+		</div>
+		<div css={buttonContainerCss}>
+			<ThemeProvider theme={buttonThemeReaderRevenueBrand}>
+				<Button
+					cssOverrides={buttonCentredCss}
+					onClick={handleDiscountOfferClick}
+				>
+					Keep support with discount
+				</Button>
+			</ThemeProvider>
+		</div>
+	</Stack>
+);
+
+export const ThankYouOffer = () => {
+	const navigate = useNavigate();
+	const cancellationContext = useContext(
+		CancellationContext,
+	) as CancellationContextInterface;
+	const productDetail = cancellationContext.productDetail;
+
+	const location = useLocation();
+	const routerState = location.state as CancellationRouterState;
+
+	if (!productDetail) {
+		return <Navigate to="/" />;
+	}
+
+	const supportStartYear = dateString(
+		new Date(productDetail.joinDate),
+		'yyyy',
+	);
+
+	const hasDiscountLive = false; // ToDo
+	const hasPaymentFailure = !!productDetail.alertText;
+	const eligibleForDiscount = !hasDiscountLive && !hasPaymentFailure;
+
+	return (
+		<section
+			css={css`
+				margin-top: ${space[4]}px;
+			`}
+		>
+			<Stack space={6}>
+				<h2
```

Comment:
Should this be an `<h1>`? I don't see an `<h1>` element prior to this `<h2>` element

Reply:
the heading outside of the page, ie "Cancel subscription" is an `h1`